src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/annotation/packageannotation/Example1.java

```diff
@@ -6,6 +6,7 @@
 </module>
 */
 
+@Deprecated
 package com.puppycrawl.tools.checkstyle.checks.annotation.packageannotation;
```

Comment:
I pushed extra commit to see if this compilation problem from javac or IDEA compiler

Reply:
ok I see compile error:
```
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /home/rivanov/java/github/romani/checkstyle/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/annotation/packageannotation/Example1.java:[9,1] package annotations should be in file package-info.java
[ERROR] /home/rivanov/java/github/romani/checkstyle/src/test/resources/com/puppycrawl/tools/checkstyle/checks/annotation/packageannotation/package-info.java:[7,1] package com.puppycrawl.tools.checkstyle.checks.annotation.packageannotation has already been annotated
[INFO] 2 errors 

```

Reply:
https://checkstyle.org/checks/annotation/packageannotation.html#Description

> For Java SE8 and above, placement of package annotations in the package-info.java file is enforced by the compiler and this check is not necessary.

ok we need to put file in non-compileable folder of examples.